_includes/utilities/html/head.html

```diff
@@ -15,7 +15,12 @@
         https://fonts.gstatic.com/;
       style-src
         'self'
-        https://fonts.googleapis.com;
+        https://fonts.googleapis.com 
+        'sha256-OTeu7NEHDo6qutIWo0F2TmYrDhsKWCzrUgGoxxHGJ8o=' 
```

Comment:
Would you please, and document in the README and/or near unsafe-inline that this assumes that we disallow HTML content from contributed datasets using strip_tags before every markdownify? So that other maintainers can maintain security in the future